### PR TITLE
Author improvements

### PR DESCRIPTION
**Pull request by jameshadfield**
This PR exports more author information in the meta JSON for display in auspice.

It introduces an (optional) `title` field in the `prepare` `config` file.

Eventually we can remove the author information from the nodes in the tree JSON, however this would break auspice and so should be done at a later date.

### CODE REVIEW

File: base/auspice_export.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                    label_str = region+'_'+str(clade)
+                    freq_json[label_str] = process_freqs(self.tree_frequencies[region][node.clade])
+        # write to one frequency json
+        if hasattr(self, 'tree_frequencies') or hasattr(self, 'mutation_frequencies'):
+            write_json(freq_json, prefix+'_frequencies.json', indent=indent)
+    else:
+        self.log.notify("Cannot export frequencies - pivots do not exist")
+
+
+def summise_publications_from_tree(tree):
```
Comment (huddlej) [inline base/auspice_export.py]: Should this be named `summarise_publications_from_tree ` instead? Or `summarize_publications_from_tree `, if you want to Americanize it.

File: zika/zika.process.py [hunk 2]
```diff
@@ -46,5 +46,4 @@
     runner.build_tree()
     runner.timetree_setup_filter_run()
     runner.run_geo_inference()
-    runner.save_as_nexus()
```
Comment (huddlej) [inline zika/zika.process.py]: Is there a reason to not save the Zika tree as a nexus file anymore or was this a deletion for debugging purposes?

File: base/auspice_export.py (hunk 1 above)
Comment (jameshadfield) [inline base/auspice_export.py]: fixed in 0301b642940cf64693fb839fc5ff82dcf629c946

File: zika/zika.process.py (hunk 2 above)
Comment (jameshadfield) [inline zika/zika.process.py]: i've reinstated this in 0301b642940cf64693fb839fc5ff82dcf629c946

There's now a script to turn the `tree.json` into a nexus file, with options such as temporal vs substitution branch lengths & the ability to resolve polytomies, which may be more useful to people.

**Comment by jameshadfield** [comment]
I think this is good to go into `master`. 

Note that this increases the size of the JSONs but this is necessary until auspice PR 427 is merged in. Once this happens, i'll remove unnecessary fields from `tree.json` and `meta.json`

**Pull request merged**